data/prefixes.js

```diff
@@ -362,6 +362,15 @@ f(require('caniuse-lite/data/features/css-placeholder'), browsers => {
   })
 })
 
+// Placeholder-shown selector
+f(require('caniuse-lite/data/features/css-placeholder-shown'), browsers => {
+  prefix([':placeholder-shown'], {
+    selector: true,
+    feature: 'css-placeholder-shown',
+    browsers
```

Comment:
css-placeholder uses `browsers: browsers.concat(['ie 10 old', 'ie 11 old', 'firefox 18 old'])` here. I don't understand what is 'ie 10 old' vs 'ie 10'.

Reply:
On my machine it is:

```
    [
      'firefox 4',  'firefox 5',  'firefox 6',
      'firefox 7',  'firefox 8',  'firefox 9',
      'firefox 10', 'firefox 11', 'firefox 12',
      'firefox 13', 'firefox 14', 'firefox 15',
      'firefox 16', 'firefox 17', 'firefox 18',
      'firefox 19', 'firefox 20', 'firefox 21',
      'firefox 22', 'firefox 23', 'firefox 24',
      'firefox 25', 'firefox 26', 'firefox 27',
      'firefox 28', 'firefox 29', 'firefox 30',
      'firefox 31', 'firefox 32', 'firefox 33',
      'firefox 34', 'firefox 35', 'firefox 36',
      'firefox 37', 'firefox 38', 'firefox 39',
      'firefox 40', 'firefox 41', 'firefox 42',
      'firefox 43', 'firefox 44', 'firefox 45',
      'firefox 46', 'firefox 47', 'firefox 48',
      'firefox 49', 'firefox 50', 'ie 10',
      'ie 11',      'kaios 2.5'
    ]
```

Reply:
We use ` old` postfix in flexbox and some other prefixes with 2 syntaxes for the same prefix